### PR TITLE
Module tutorial

### PR DESCRIPTION
**Pull request by ProgVal**

### CODE REVIEW

File: development-environment.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Setting up a development environment
+
+This guide will show you how to install a WebUI and a core on your
+computer and configure them to test your modules locally.
+
+TL;DR version: just copy-past the code blocks at the right place.
+
+As you may want to connect your instance with other modules, we
+will connect it to the Platypus, the official demo of the PPP.
```
Comment (Tpt) [inline development-environment.md]: It looks like it is not done in the config given in this tutorial

File: python-module-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+Note that you have to install `python-memcache` and to run a memcached
+instance on your computer.
+It may be a good idea to make the timeout and the IP address of the
+cache server configurable. But you should already know how to make
+your module configurable (otherwise, see the section above), so
+this is left as an exercice to the reader.
+
+## Enriching a resource
+
+TODO (not specificied yet)
```
Comment (Tpt) [inline python-module-tutorial.md]: typo

**Comment by Ezibenroc** [comment]
> Implement the constructor this way:
```python
def __init__(self, request):
    self.request = request
```

Why don't we generate this code automatically with the script?

**Comment by ProgVal** [comment]
indeed, we should do that

**Comment by Ezibenroc** [comment]
>
```python
return [shortcuts.build_answer(self.request, tree, {}, 'OSM')]
```

Maybe we should explain the meaning of this (at least, what are the arguments).

**Comment by Ezibenroc** [comment]
Ok for the pull request, great job!

**Comment by Tpt** [comment]
Yeah! \o/

**Comment by yhamoudi** [comment]
> great job!

:thumbsup:

**Pull request merged**